javascript/sdk/lib/types.ts

```diff
@@ -391,6 +412,11 @@ export interface DeployParams {
   hostIds: string[];
 }
 
+export interface DeployedEndpoints {
+  id: string;
```

Comment:
fyi - this id is the enum value of the Partner (1 for crowdstrike, etc). Know you added that ControlCode enum somewhere, not sure if you want to tie in here in some way?

Reply:
updated!